packages/config/src/projects/layer2s/kinto.ts

```diff
@@ -105,7 +109,26 @@ export const kinto: Layer2 = orbitStackL2({
         sinceTimestamp: new UnixTime(1702607855),
       },
     },
+    {
+      uses: [
+        { type: 'liveness', subtype: 'batchSubmissions' },
+        { type: 'l2costs', subtype: 'batchSubmissions' },
+      ],
+      query: {
+        // tracks both blobs and calldata (works for op stack, does it work here too?)
+        formula: 'transfer',
```

Comment:
why not tracking addSequencerL2BatchFromBlobs? i dont understand

Reply:
i am trying out this simpler logic because we are successfully using it in the op stack template and i thought it might work here too? If it does i will add it to parallel too

Reply:
i mean, we don't want to count random txs unrelated to that function if sent from that address. I know that it's unlikely, but maybe not? i don't see why we should add random assumptions. In OP stack is that way because they are literally transfers to EOA